osu.Game.Rulesets.Osu.Tests/Editor/TestSceneSliderVelocityAdjust.cs

```diff
@@ -29,7 +30,7 @@ public partial class TestSceneSliderVelocityAdjust : OsuGameTestScene
 
         private Slider? slider => editorBeatmap.HitObjects.OfType<Slider>().FirstOrDefault();
 
-        private TimelineHitObjectBlueprint blueprint => editor.ChildrenOfType<TimelineHitObjectBlueprint>().FirstOrDefault()!;
+        private TimelineHitObjectBlueprint blueprint => editor.ChildrenOfType<TimelineHitObjectBlueprint>().FirstOrDefault(b => b.Item.GetEndTime() != b.Item.StartTime)!;
```

Comment:
what's this?

Reply:
with seasonal enabled, this test uses the new christmas beatmap which features a hitobject earlier than expected.

this change allows it to still pass by finding the first slider

Reply:
That's pretty dire. Reading the test it seems to be implicitly expected that the `blueprint` matches the `slider`, so I'd probably just do that explicitly (untested, but I'd hope it works):

```suggestion
        private TimelineHitObjectBlueprint blueprint => editor.ChildrenOfType<TimelineHitObjectBlueprint>().FirstOrDefault(b => b.Item == slider)!;
```

Reply:
I can undo this with the `Loader` change, in fact. But I need to figure out how to make that coexist with the intro test scene.